### PR TITLE
Add badge field for online and vm-iso collections

### PR DESCRIPTION
**Pull request by kingthorin**
- offline > Fix/restore missing brace.
- online > Add badge field and populate where applicable.
- vm-iso > Add badge field and populate where applicable.
- README > Tweak to be applicable for current published content.

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,7 @@
 
 The OWASP Vulnerable Web Applications Directory Project (VWAD, https://owasp.org/www-project-vulnerable-web-applications-directory/) is a comprehensive and well maintained registry of all known vulnerable web applications currently available.
 
-The three individual lists are avilable via:
-- https://owasp.org/www-project-vulnerable-web-applications-directory/online 
-- https://owasp.org/www-project-vulnerable-web-applications-directory/offline
-- https://owasp.org/www-project-vulnerable-web-applications-directory/vm-iso
+The three individual lists are available via separate tabs on: 9https://owasp.org/www-project-vulnerable-web-applications-directory/)[https://owasp.org/www-project-vulnerable-web-applications-directory/]
```
Comment (thc202) [inline README.md]: The Markdown for the link is not correct.

File: src/data/online.json [hunk 2]
```diff
@@ -368,30 +396,33 @@
...
+		"notes": "(testuser/testpass)",
+		"badge": null
 	},
 	{
 		"url": "http://blasze.com/xsstestsuite/",
 		"name": "XSS Test Suite",
 		"technology": [],
 		"references": [],
 		"author": null,
-		"notes": null
```
Comment (thc202) [inline src/data/online.json]: This was removed.

**Comment by kingthorin** [comment]
Tks @thc202 will fix that shortly

File: README.md (hunk 1 above)
Comment (kingthorin) [reply README.md]: Fixed

File: src/data/online.json (hunk 2 above)
Comment (kingthorin) [reply src/data/online.json]: Restored

File: README.md (hunk 1 above)
Comment (thc202) [reply README.md]: The brackets are swapped though.

Comment (kingthorin) [reply README.md]: Ya at first I thought it was just the 9, but then i realized i had it backward.

**Comment by kingthorin** [comment]
Rebased current to pickup the JSON lint action.

**Pull request merged**